server/router/routes/domain-list-handler.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Uber Technologies Inc.
```

Comment:
moved `server/router.js` route handlers into separate files in `server/router/routes`